eclipse/eclipse-code-formatter.xml

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We reference this [here](https://github.com/spring-cloud/spring-cloud-netflix#code-conventions-and-housekeeping) and have [one in spring-cloud-build](https://github.com/spring-cloud/spring-cloud-build/blob/master/spring-cloud-build/eclipse-code-formatter.xml).  Maybe the docs should link to that one if they don't already.